cmd/docker/docker.go

```diff
@@ -57,6 +57,7 @@ func newDockerCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags = cmd.Flags()
 	flags.BoolVarP(&opts.Version, "version", "v", false, "Print version information and quit")
 	flags.StringVar(&opts.ConfigDir, "config", cliconfig.Dir(), "Location of client config files")
+	flags.BoolVarP(&opts.TLSGetPass, "tlsgetpass", "", cliconfig.TLSGetPass(), "Request passphrase if needed to decrypt TLS keys")
```

Comment:
I'm skeptical that this needs a flag. The cert should be stored in a format the allows us to detect if it's encrypted rather than require the user to know and inform the CLI that it should request a passphrase.